src/-helpers.js

```diff
@@ -7,10 +6,16 @@ var classTest = /^\s*class\s+/;
 var helpers = {
 	assignEverything: function(d, s) {
 		Object.getOwnPropertyNames(s).concat(Object.getOwnPropertySymbols(s)).forEach(function(key) {
-			Object.defineProperty(d,key, Object.getOwnPropertyDescriptor(s,key));
+			Object.defineProperty(d, key, Object.getOwnPropertyDescriptor(s, key));
 		});
 		return d;
 	},
+	isBuiltInButNotArrayOrPlainObjectOrElement: function(obj) {
+		if (obj instanceof Element) {
```

Comment:
Should be [`getGlobal().Element`](https://canjs.com/doc/can-globals/global/global.html) if we want this to work in ssr (at least not throw).